app.js

```diff
@@ -46,7 +50,7 @@ var isAuthenticated = function(req, res, next) {
   res.redirect('/login');
 };
 
-app.get('/', isAuthenticated, function(req, res) {
+app.get('/', function(req, res) {
```

Comment:
Removed the isAuthenticated middleware for testing. Add it back if you want to test login functionality.